uber/config.py

```diff
@@ -503,7 +507,7 @@ def _is_intstr(s):
 c.MERCH_SHIRT_OPTS = [(c.SIZE_UNKNOWN, 'select a size')] + list(c.PREREG_SHIRT_OPTS)
 c.DONATION_TIER_OPTS = [(amt, '+ ${}: {}'.format(amt, desc) if amt else desc) for amt, desc in c.DONATION_TIER_OPTS]
 
-c.DONATION_TIER_DESCRIPTIONS = _config.get('donation_tier_descriptions', [])
+c.DONATION_TIER_DESCRIPTIONS = _config.get('donation_tier_descriptions', {})
```

Comment:
This is an unrelated fix.  If you look at the line below this one, we're expected ``c.DONATION_TIER_DESCRIPTIONS`` to always be a dict, so in my local Vagrant it was blowing up because I didn't have it defined.